Ccount.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h> 
+int main(){
+char string [500], ch;
+int x, count = 0;
+
+printf("Enter a string of input characters: ");
+scanf("%s", string);
+
+for(x = 0; x <= 500; x++) {			/*for loop to test each character in the string*/
+	ch = string[x];
```

Comment:
Your logic is slightly off - if you input the word "cAT", it tells me there are two Cs. looks like it's something to do with your character conversion.